spec/ysh-methods.test.sh

```diff
@@ -393,6 +393,15 @@ pp line (en2fr => values())
 (List)   ["bonjour","ami","chat"]
 ## END
 
+#### Dict -> erase()
+var book = {title: "The Histories", author: "Herodotus"}
+call book->erase("author")
```

Comment:
I would add another call to test that we do nothing when the key doesn't exist

Reply:
New commit has the extra call.

Doing nothing when the key doesn't exist has really grown on me. Wasn't sure about it at first (felt like a silent failure) but realised that a lot of time I want a dictionary in a particular way and don't care whether the key was there or not, as long as it isn't now.